src/main/kotlin/net/leanix/vsm/sbomBooster/service/OrtService.kt

```diff
@@ -97,4 +101,18 @@ class OrtService(
         val folder = Paths.get("tempDir", downloadFolder).toFile()
         folder.deleteRecursively()
     }
+
+    private fun setupOutput(projectUrl: String, phase: String, processBuilder: ProcessBuilder) {
```

Comment:
If I get it right, you're getting a file content to memory:
Paths.get(
                "tempDir",
                "${projectUrl.substringAfterLast("/")}_$phase.txt"
                
Then we save it to a file:
            ).toFile()
then you save it somewhere else:
       FileOutputStream(repoFileName)
       
   Not sure if I understood 😆

Reply:
The `tempDir` is the name that is used from docker for the mounted volume that you pass with the command. 

```
docker run --pull=always --rm \
           -v /var/run/docker.sock:/var/run/docker.sock \
           -v <temp-folder-to-be-used-for-storing-data>:/tempDir \
           -e MOUNTED_VOLUME='<temp-folder-to-be-used-for-storing-data>' \
           -e LEANIX_HOST='<leanix-workspace-host>' \
           -e LEANIX_TOKEN='<leanix-technical-user-token>' \
           -e GIT_PROVIDER='GITHUB' \
           -e GITHUB_TOKEN='<github-token>' \
           -e GITHUB_ORGANIZATION='<github-organization>' \
           leanixacrpublic.azurecr.io/vsm-sbom-booster
```

Using get with `tempDir` and then applying the file name, you get the full path for the file to be saved.
I had ti use the trial and error method to make it work as expected.

Reply:
Docker has access to the host file system only through the mounted volumes.